frontend/src/metabase/visualizations/echarts/cartesian/waterfall/model/index.ts

```diff
@@ -97,28 +97,11 @@ function getSortedAggregatedRows(
   return aggregatedRows;
 }
 
-function getWaterfallNegativeTranslation(
```

Comment:
I moved the `runningSum` and `negativeTranslation` computation to within `getWaterfallDataset`. It made more sense for them to be there, because `runningSum` is needed to create the `waterfallDataset`, but isn't a part of the chart model itself because it's not used anywhere else.